docassemble/SMPDecisionTree/data/questions/software.yml

```diff
@@ -396,612 +366,12 @@ fields:
-    - 'Vim (Vim License)': 'Vim (Vim License)'
-    - 'VOSTROM (VOSTROM Public License for Open Source)': 'VOSTROM (VOSTROM Public License for Open Source)'
-    - 'VSL-1.0 (Vovida Software License v1.0)': 'VSL-1.0 (Vovida Software License v1.0)'
-    - 'W3C (W3C Software Notice and License (2002-12-31))': 'W3C (W3C Software Notice and License (2002-12-31))'
-    - 'W3C-19980720 (W3C Software Notice and License (1998-07-20))': 'W3C-19980720 (W3C Software Notice and License (1998-07-20))'
-    - 'W3C-20150513 (W3C Software Notice and Document License (2015-05-13))': 'W3C-20150513 (W3C Software Notice and Document License (2015-05-13))'
-    - 'w3m (w3m License)': 'w3m (w3m License)'
-    - 'Watcom-1.0 (Sybase Open Watcom Public License 1.0)': 'Watcom-1.0 (Sybase Open Watcom Public License 1.0)'
-    - 'Widget-Workshop (Widget Workshop License)': 'Widget-Workshop (Widget Workshop License)'
-    - 'Wsuipa (Wsuipa License)': 'Wsuipa (Wsuipa License)'
-    - 'WTFPL (Do What The F*ck You Want To Public License)': 'WTFPL (Do What The F*ck You Want To Public License)'
-    - 'X11 (X11 License)': 'X11 (X11 License)'
-    - 'X11-distribute-modifications-variant (X11 License Distribution Modification Variant)': 'X11-distribute-modifications-variant (X11 License Distribution Modification Variant)'
-    - 'Xdebug-1.03 (Xdebug License v 1.03)': 'Xdebug-1.03 (Xdebug License v 1.03)'
-    - 'Xerox (Xerox License)': 'Xerox (Xerox License)'
-    - 'Xfig (Xfig License)': 'Xfig (Xfig License)'
-    - 'XFree86-1.1 (XFree86 License 1.1)': 'XFree86-1.1 (XFree86 License 1.1)'
-    - 'xinetd (xinetd License)': 'xinetd (xinetd License)'
-    - 'xkeyboard-config-Zinoviev (xkeyboard-config Zinoviev License)': 'xkeyboard-config-Zinoviev (xkeyboard-config Zinoviev License)'
-    - 'xlock (xlock License)': 'xlock (xlock License)'
-    - 'Xnet (X.Net License)': 'Xnet (X.Net License)'
-    - 'xpp (XPP License)': 'xpp (XPP License)'
-    - 'XSkat (XSkat License)': 'XSkat (XSkat License)'
-    - 'YPL-1.0 (Yahoo! Public License v1.0)': 'YPL-1.0 (Yahoo! Public License v1.0)'
-    - 'YPL-1.1 (Yahoo! Public License v1.1)': 'YPL-1.1 (Yahoo! Public License v1.1)'
-    - 'Zed (Zed License)': 'Zed (Zed License)'
-    - 'Zeeff (Zeeff License)': 'Zeeff (Zeeff License)'
-    - 'Zend-2.0 (Zend License v2.0)': 'Zend-2.0 (Zend License v2.0)'
-    - 'Zimbra-1.3 (Zimbra Public License v1.3)': 'Zimbra-1.3 (Zimbra Public License v1.3)'
-    - 'Zimbra-1.4 (Zimbra Public License v1.4)': 'Zimbra-1.4 (Zimbra Public License v1.4)'
-    - 'Zlib (zlib License)': 'Zlib (zlib License)'
-    - 'zlib-acknowledgement (zlib/libpng License with Acknowledgement)': 'zlib-acknowledgement (zlib/libpng License with Acknowledgement)'
-    - 'ZPL-1.1 (Zope Public License 1.1)': 'ZPL-1.1 (Zope Public License 1.1)'
-    - 'ZPL-2.0 (Zope Public License 2.0)': 'ZPL-2.0 (Zope Public License 2.0)'
-    - 'ZPL-2.1 (Zope Public License 2.1)': 'ZPL-2.1 (Zope Public License 2.1)'
+    - 'To be determined': 'To be determined'
```

Comment:
- Apache 2.0 is still at the top of the list, no worries 😉 
- My guess would be that if you choose a relatively non-standard license, you're advanced/aware enough to paste the SPDX identifier in the field to make it machine readable? Although saying that, it would make sense to notify the user of that being the right thing to do here.

Reply:
- Ah, the diff view cut it out and I didn't check in the original file. Good point.
- Pasting the SPDX identifier would not be my personal first instinct. I might write the human-readable part from memory, which might or might not match the "official" description and would probably not be machine-readable.
- I don't think people would scroll through the licenses like a shopping list and pick one they like. I hope the users would pick a license, and then find it in the list afterwards. Personally I do find the search bar intuitive, but I don't know if others see it the same way. The benefit of the long list is that it's exhaustive, so you're guaranteed to end up with a SPDX identifier if you don't choose "Other" or "TBD". I'm up for scrapping the text descriptions, but are the license choices really intimidating? This is meant as an actual, not rhetorical question, and I'm up for debating this further. :slightly_smiling_face: